source/common/common/utility.h

```diff
@@ -709,7 +709,6 @@ template <class Value> class TrieLookupTable {
       children_[child_index - min_child_] = std::move(node);
     }
 
-  private:
```

Comment:
nit: move the `Value value_{};` field from above here (or move these fields up to the beginning of the class).

Reply:
Done.